src/status_im2/contexts/wallet/common/network_preferences/view.cljs

```diff
@@ -4,59 +4,67 @@
             [quo.foundations.resources :as resources]
             [quo.theme :as quo.theme]
             [status-im2.contexts.wallet.common.network-preferences.style :as style]
-            [utils.i18n :as i18n]))
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
 
-(def mainnet
+(defn mainnet
+  [customization-color]
   [{:title        "Mainnet"
```

Comment:
should we i18n "Mainnet" ? seems likely we should.

Reply:
I'm not sure whether the network names have translations. This is an existing code in the network preferences sheet. I will check and update it if required.

Reply:
Sure we can do a follow up for any details like this. The pr has enough going already :)